src/postgraphile/cli.ts

```diff
@@ -11,10 +11,13 @@ import { Pool } from 'pg'
 import cluster = require('cluster')
 import { makePluginHook, PostGraphilePlugin } from './pluginHook'
 import debugFactory = require('debug')
+import { Postgraphile } from '../interfaces'
+import mixed = Postgraphile.mixed
 
 const debugCli = debugFactory('postgraphile:cli')
 
-// tslint:disable no-console
+// tslint:disable no-console no-shadowed-variable
+// no-shadowed-variable is erroroneously reporting
```

Comment:
im betting the older tslint didnt do this :/  it was claiming that `plugins` in a typedef was shadowed.  infact, it was just annotating the type.  bonkers AST parse.  as you can see, i didn't modify anything to do w/ a variable called `plugins` in this file.